pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/PinotConnection.java

```diff
@@ -70,10 +66,10 @@ public class PinotConnection extends AbstractBaseConnection {
     }
     _session = new org.apache.pinot.client.Connection(properties, brokers, transport);
 
-    for (String possibleQueryOption: POSSIBLE_QUERY_OPTIONS) {
-      Object property = properties.getProperty(possibleQueryOption);
-      if (property != null) {
-        _queryOptions.put(possibleQueryOption, parseOptionValue(property));
+    for (Map.Entry<Object, Object> property : properties.entrySet()) {
+      String configKey = QueryOptionsUtils.getConfigKey(property.getKey());
+      if (property.getKey().equals(configKey)) {
```

Comment:
This check would prevent case insensitivity

Reply:
done by just null check.